azurerm/internal/services/monitor/tests/resource_arm_monitor_action_rule_test.go

```diff
@@ -0,0 +1,650 @@
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("bad: Get on AlertsManagement ActionRulesClient: %+v", err)
+			}
+		}
+		return nil
+	}
+	return nil
+}
+
+func testAccAzureRMAlertsManagementActionRule_diagnostics_basic(data acceptance.TestData) string {
+	template := testAccAzureRMAlertsManagementActionRule_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_monitor_action_rule" "test" {
+  name                = "acctest-ar-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  type                = "Diagnostics"
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMAlertsManagementActionRule_diagnostics_requiresImport(data acceptance.TestData) string {
+	template := testAccAzureRMAlertsManagementActionRule_diagnostics_basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_monitor_action_rule" "import" {
+  name                = azurerm_monitor_action_rule.test.name
+  resource_group_name = azurerm_monitor_action_rule.test.resource_group_name
+  type                = "Diagnostics"
+
+  scope {
+    type         = "ResourceGroup"
+    resource_ids = [azurerm_resource_group.test.id]
+  }
+}
+`, template)
+}
+
+func testAccAzureRMAlertsManagementActionRule_diagnostics_complete(data acceptance.TestData) string {
+	template := testAccAzureRMAlertsManagementActionRule_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_monitor_action_rule" "test" {
+  name                = "acctest-ar-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  type                = "Diagnostics"
+  enabled             = false
+  description         = "actionRule-test"
+
+  scope {
+    type         = "ResourceGroup"
+    resource_ids = [azurerm_resource_group.test.id]
+  }
+
+  condition {
+    alert_context {
+      operator = "Contains"
+      values   = ["context1", "context2"]
+    }
+
+    alert_rule_id {
+      operator = "Contains"
+      values   = ["ruleId1", "ruleId2"]
+    }
+
+    description {
+      operator = "DoesNotContain"
+      values   = ["description1", "description2"]
+    }
+
+    monitor {
+      operator = "NotEquals"
+      values   = ["Fired"]
+    }
+
+    monitor_service {
+      operator = "Equals"
+      values   = ["Data Box Edge", "Data Box Gateway", "Resource Health"]
+    }
+
+    severity {
+      operator = "Equals"
+      values   = ["Sev0", "Sev1", "Sev2"]
+    }
+
+    target_resource_type {
+      operator = "Equals"
+      values   = ["Microsoft.Compute/VirtualMachines", "microsoft.batch/batchaccounts"]
+    }
+  }
+
+  tags = {
+    ENV = "Test"
+  }
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMAlertsManagementActionRule_actionGroup_basic(data acceptance.TestData) string {
+	template := testAccAzureRMAlertsManagementActionRule_actionGroup_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_monitor_action_rule" "test" {
+  name                = "acctest-ar-%d"
```

Comment:
this is part of the monitor service so the resource gorup should be
```suggestion
  name                = "acctest-mon-%d"
```
or 
```suggestion
  name                = "acctest-moniter-%d"
```